pkg/discovery/globalnet/globalnet.go

```diff
@@ -408,8 +408,10 @@ func ValidateExistingGlobalNetworks(config *rest.Config, namespace string) error
 	}
 
 	if globalnetInfo != nil {
-		if err = IsValidCIDR(globalnetInfo.GlobalnetCidrRange); err != nil {
-			return fmt.Errorf("invalid GlobalnetCidrRange: %s", err)
+		if globalnetInfo.GlobalnetEnabled {
```

Comment:
You can combine the if statements, ie:

`if globalnetInfo != nil && globalnetInfo.GlobalnetEnabled {`

Reply:
yes, you are right. I will fix it.